tests/accuracy_tests/sim_for_alphapeel_accu_test/sim_for_metafounder_accu.R

```diff
@@ -0,0 +1,267 @@
+  parentsA <- selectInd(pop = popA, nInd = 10, trait = 1)
+  parentsB <- selectInd(pop = popB, nInd = 10, trait = 2)
+  if (generation == nGen){
+    popA <- randCross(pop = parentsA, nCrosses = 75)
+    popB <- randCross(pop = parentsB, nCrosses = 75)
+  } else {
+    popA <- randCross(pop = parentsA, nCrosses = 100)
+    popB <- randCross(pop = parentsB, nCrosses = 100)
+  }
+  
+  popA <- setPheno(pop = popA, varE = diag(varE))
+  popB <- setPheno(pop = popB, varE = diag(varE))
+  data <- collectData(pop = popA, data = data, population = "A", generation = generation)
+  data <- collectData(pop = popB, data = data, population = "B", generation = generation)
+}
+
+# Continued selection on each trait in each separate population,
+# but add also continually admixed population selected on an index
+popAB <- randCross(pop = c(parentsA, parentsB), nCrosses = 50)
+popAB <- setPheno(pop = popAB, varE = diag(varE))
+data <- collectData(pop = popAB, data = data, population = "AB", generation = generation)
+economicWeights <- c(1, 1)
+selIndexWeights <- smithHazel(econWt = economicWeights, varG = varG, varP = varG + varE)
+for (generation in 6:10) {
+  parentsA <- selectInd(pop = popA, nInd = 10, trait = 1)
+  parentsB <- selectInd(pop = popB, nInd = 10, trait = 2)
+  parentsAB <- selectInd(pop = popAB, nInd = 6, trait = selIndex, scale = TRUE, b = selIndexWeights)
+  parentsA4AB <- selectInd(pop = popA, nInd = 2, trait = selIndex, scale = TRUE, b = selIndexWeights)
+  parentsB4AB <- selectInd(pop = popB, nInd = 2, trait = selIndex, scale = TRUE, b = selIndexWeights)
+  parentsAB <- c(parentsAB, parentsA4AB, parentsB4AB)
+  popA <- randCross(pop = parentsA, nCrosses = 75)
+  popB <- randCross(pop = parentsB, nCrosses = 75)
+  popAB <- randCross(pop = parentsAB, nCrosses = 50)
+  popA <- setPheno(pop = popA, varE = diag(varE))
+  popB <- setPheno(pop = popB, varE = diag(varE))
+  popAB <- setPheno(pop = popAB, varE = diag(varE))
+  data <- collectData(pop = popA,  data = data, population = "A",  generation = generation)
+  data <- collectData(pop = popB,  data = data, population = "B",  generation = generation)
+  data <- collectData(pop = popAB, data = data, population = "AB", generation = generation)
+}
+
+data$pedigree$population <- factor(data$pedigree$population, levels = c("A", "B", "AB"))
+summary(data$pedigree$population)
+data$pedigree$gv <- c(selIndex(Y = as.matrix(data$pedigree[, c("gv1", "gv2")]), scale = TRUE, b = selIndexWeights))
+data$pedigree$pv <- c(selIndex(Y = as.matrix(data$pedigree[, c("pv1", "pv2")]), scale = TRUE, b = selIndexWeights))
+
+data$pedigree$generationPlotShift <- data$pedigree$generation +
+  c(-0.25, +0.25, 0)[data$pedigree$population]
+
+# Get pedigree and assign metafounders
+pedigree <- data.frame(data$pedigree$id, data$pedigree$mid, data$pedigree$fid, data$pedigree$population, data$pedigree$generation)
+colnames(pedigree) <- c("id", "mid", "fid", "population", "generation")
+pedigree <- pedigree[c(401:1400),]
```

Comment:
We have hardcoded numbers here but nGen comes into this script externally - will that clash with these hardcoded numbers here?